recipes-connectivity/networkmanager/networkmanager_%.bbappend

```diff
@@ -0,0 +1,15 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+
+SRC_URI += " \
+	file://0001-wwan-fix-provision-of-IPv6-addresses.patch \
```

Comment:
is upstream aware of this problematic? is there a plan to integrate this patch?
is this really a fix? as far as i understand the patch disables ipv6?

Reply:
yes, the patch is really necessary if we want to use IPv6 addresses as propagated by broadband provider, I tested this.
I found it in the pull request https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/merge_requests/1170 which didn't make it into the code at first.
However, beginning from version 1.38 this patch is included as of commit https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/commit/586afdfc5298285d09a5b4965690899833a21abb (part of version 1.37.3-dev-52-g586afdfc52)